RepClient/src/main.ts

```diff
@@ -1,6 +1,5 @@
 import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import 'hammerjs';
```

Comment:
Is this touch gesture package redundant now?

Reply:
No, this is how you used to use hammerjs prior to Angular 9 after angular 9 you instead use Hammerjs Module and import it that way. Hammerjs is still the default gesture control for angular see https://github.com/angular/angular/issues/35547#issuecomment-588277881 from angular team